README.md

```diff
@@ -25,29 +18,16 @@ users to be on certain machines.
 The following attributes are required for each user:
 
 * username - The user's username.
-* name - The full name of the user (gecos field)
-* uid - The numeric user id for the user. This is required for uid consistency
```

Comment:
Since you've added some special logic around the presence or absence of this var, you should explain that in the doc.